spec/fdk-harvest-adm.yaml

```diff
@@ -0,0 +1,148 @@
+                type: string
+              description: url to the new data source
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/DataSource"
+    get:
+      tags:
+        - dataSource
+      summary: Query for data sources.
+      description: Returns a collection of matching data sources
+      operationId: getDataSources
+      parameters:
+        - name: name
+          in: query
+          description: A query string to match a data source name
+          required: false
+          schema:
+            type: string
+        - name: publisher
+          in: query
+          description: If you want to filter by organizationId of the publisher
+          required: false
+          schema:
+            type: string
+        - name: dataSourceType
+          in: query
+          description: If you want to filter by the data source type
+          required: false
+          schema:
+            type: string
+            enum:
+              - SKOS-AP-NO
+              - DCAT-AP-NO
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/DataSource"
+  /datasources/{id}:
+    get:
+      tags:
+        - dataSource
+      description: Get a specific data source by id
+      operationId: getDataSourceById
+      parameters:
+        - name: id
+          in: path
+          description: id
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DataSource"
+    put:
+      tags:
+        - dataSource
+      description: Update a specific dataSource by id
+      operationId: updateDataSource
+      parameters:
+        - name: id
+          in: path
+          description: id
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DataSource"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/DataSource"
+components:
+  schemas:
+    DataSource:
+      type: object
+      description: A data source that is to be harvester
+      properties:
+        id:
+          type: string
+          description: system-generated id (GUID)
+        dataSourceType:
+          type: string
+          enum:
+            - SKOS-AP-NO
+            - DCAT-AP-NO
+        url:
+          type: string
+          description: Link to the organization entry in The Brønnøysund Register Centre
```

Comment:
Jeg tror den beskrivelse er feil. De skulle være endepunkte url til standard interface api?

Reply:
Ja. Det skal vere "Link to the datasource to be harvested"